src/sasquatchbackpack/sasquatch.py

```diff
@@ -143,7 +144,81 @@ def __init__(
             else redis_address
         )
 
-    def create_topic(self) -> str:
+    def _get_topic_exists(self) -> bool | str:
+        """Determine whether a topic exists on kafka already.
+
+        Returns
+        -------
+        exists : bool | str
+            True if this Dispatcher's topic exists in the proper rest proxy,
+            otherwise, False. Will return a str containing relevant responses
+            in the event of an error.
+        """
+        headers = {"content-type": "application/json"}
+
+        try:
+            r = requests.get(
+                f"{self.config.sasquatch_rest_proxy_url}/v3/clusters",
+                headers=headers,
+                timeout=10,
+            )
+            r.raise_for_status()  # Raises HTTPError for bad responses
+            cluster_id = r.json()["data"][0]["cluster_id"]
+        except requests.RequestException as e:
+            return f"Error getting cluster ID: {e}"
+
+        headers = {"content-type": "application/json"}
+
+        try:
+            response = requests.get(
+                f"{self.config.sasquatch_rest_proxy_url}/v3/clusters/"
+                f"{cluster_id}/topics",
```

Comment:
In the scenario you want to check for the topic existence, you can do it via GET `/v3/clusters/{cluster_id}/topics/{topic_name}` directly. It will return code 200 if exists and a 400 if not.